assets/src/models/createDetourMachine.ts

```diff
@@ -4,6 +4,8 @@ import { ShapePoint } from "../schedule"
 export const createDetourMachine = setup({
   types: {} as {
     context: {
+      startPoint: ShapePoint | null
```

Comment:
Should replace this `null` with `undefined`.

Reply:
[Done](https://github.com/mbta/skate/pull/2629/commits/5917f4542f860a79e3cb048d771f49d908784d49)